samples/ReloadSample/Program.fs

```diff
@@ -120,13 +123,13 @@ let configureLogging (builder : ILoggingBuilder) =
 
 [<EntryPoint>]
 let main _ =
-    let contentRoot = Directory.GetCurrentDirectory()
-    let webRoot     = Path.Combine(contentRoot, "WebRoot")
+    // let contentRoot = Directory.GetCurrentDirectory()
+    let webroot     = "wwwroot"
     WebHostBuilder()
         .UseKestrel()
-        .UseContentRoot(contentRoot)
+        // .UseContentRoot(contentRoot)
```

Comment:
Setting the ContentRoot to `Directory.GetCurrentDirectory()` will end up setting it based on working directory.  I don't think this is intended.  Should we just remove this?

Reply:
```suggestion
```

yep, we can remove it. the content root is the project directory by default, and the webroot is `projectdir/wwwroot` by default as well, so technically we don't have to do either of these?

Reply:
I figured having the webroot being explicit would be easier for people to figure out how to change things if needed.